test/format/format.sh

```diff
@@ -362,6 +367,11 @@ format()
 		args=${smoke_list[$smoke_next]}
 		smoke_next=$(($smoke_next + 1))
 		echo "$name: starting smoke-test job in $dir ($(date))"
+	elif [[ $timing_stress_split_test -ne 0 ]]; then
+		args=$format_args
+		for k in {1..7}; do
+			args+=" timing_stress_split_$k=$(($RANDOM%2))"
+		done
```

Comment:
It would be helpful to add an echo line indicating the starting of the job, just as what `if` and `else` branches do. 
Also, the Jenkins job prints out the timeing_stress_split_x values being set for the format run, which would be nice to retain.

Reply:
Yes, good point, I'll add the echo line. I'm not sure how useful it is to print the `timing_stress_split_` values for every run, particularly without the rest of the config or any other information related to the test run. If the test fails, the artifacts will have the full config plus all other relevant information. But please let me know if there's something I'm missing!